layers/+lang/c-c++/packages.el

```diff
@@ -23,7 +23,7 @@
     helm-cscope
     helm-gtags
     semantic
-    (srefactor :toggle (version< emacs-version "24.4"))
+    (srefactor :toggle (not (version< emacs-version "24.4")))
```

Comment:
No it's not. `srefactor` requires Emacs to be equal to 24.4 or above, not equal to 24.4 or older.